packages.dhall

```diff
@@ -3,6 +3,71 @@ let upstream =
 
 let overrides = {=}
 
-let additions = {=}
```

Comment:
Obvious to all of us, but just noting for the record that we'll merge once we can eliminate this change.

Reply:
I think we should add a tag to PRs that are basically "good to merge" once their dependencies are included in the package set. Thoughts?

Reply:
Sounds good. Added the `package-set` label. Feel free to edit that label if you had something else in mind.